junit/InPlaceArithmaticTests.java

```diff
@@ -0,0 +1,180 @@
+package junit;
+import javatrix.Matrix;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+public class InPlaceArithmaticTests{
+    /**
+    * Test byElementInPlace(Matrix, Matrix, boolean, Operator).
+    */
+    @Test
+    public void byElementInPlaceTest1(){
+        double[][] a1 = {
+            {  1.0,   2.0,   3.0,   4.0}, 
+            {  5.0,   6.0,   7.0,   8.0}, 
+            {  9.0,  10.0,  11.0,  12.0}
+        };
+        Matrix m1 = new Matrix(a1);
+        double[][] a2 = {
+            { -3.0,  -4.0,  -5.0,  -6.0}, 
+            { -7.0,  -8.0,  -9.0, -10.0}, 
+            {-11.0, -12.0, -13.0, -14.0}
+        };
+        Matrix m2 = new Matrix(a2);
+        double[][] answer = {
+            { -2.0, -2.0, -2.0, -2.0},
+            { -2.0, -2.0, -2.0, -2.0},
+            { -2.0, -2.0, -2.0, -2.0}
+        };
+        m1.byElement(m2, true, (a, b) -> a + b);
```

Comment:
So, we're not running checkstyle on these other test files, are we? I imagine that this would fail because of the lambda, no?